scripts/setSignerVersion.mts

```diff
@@ -12,15 +13,15 @@ const { values: { [VERSION_PARAM_NAME]: version = '' }} = parseArgs({
   }
 })
 
-assert(/\d\.\d\.\d/.test(version), `The "${VERSION_PARAM_NAME}" argument: "${version}" does not match the "x.x.x" format.`)
+assert(/^\d+\.\d+\.\d+(-alpha.\d+)?$/.test(version), `The "${VERSION_PARAM_NAME}" argument: "${version}" does not match the "x.x.x" format.`)
```

Comment:
it doesn't really matter... I'm sorry... I can't help it, I'm just this way...
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/bdd3188c-dba5-439c-a99d-800d039ccb8c)

If you could update the `does not match the "x.x.x" format.`

Reply:
Good good, valid point with the regex 👍 - my overlook.